src/__tests__/checkDockerInstallation.test.ts

```diff
@@ -0,0 +1,48 @@
+import { checkDockerInstallation } from "../actions/checkDockerInstallation";
+import { Logger } from "../utils/logger";
+import { execCommand } from "../utils/execCommand";
+
+jest.mock("../utils/logger");
+jest.mock("../utils/execCommand");
+
+describe("checkDockerInstallation", () => {
+  const mockExecCommand = execCommand as jest.Mock;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should log that Docker is already installed if command returns true", async () => {
+    mockExecCommand.mockResolvedValue("docker");
+
+    await checkDockerInstallation();
+
+    expect(Logger.info).toHaveBeenCalledWith("Docker is already installed.");
+  });
+
+  it("should attempt to install Docker if not installed", async () => {
+    mockExecCommand.mockResolvedValue("");
+
+    await checkDockerInstallation();
+
+    expect(Logger.info).toHaveBeenCalledWith(
+      "Docker is not installed. Installing Docker..."
+    );
+    expect(execCommand).toHaveBeenCalledWith(
+      "curl -fsSL https://get.docker.com -o get-docker.sh"
```

Comment:
maybe put the url to the top

Reply:
done in https://github.com/jaracogmbh/docker-compose-health-check-action/pull/2/commits/a358106562b9ec6fa7cb73c30e87f9aec4809ec1